src/mscorlib/src/System/Lazy.cs

```diff
@@ -254,29 +478,24 @@ internal T ValueForDebugDisplay
                 {
                     return default(T);
                 }
-                return ((Boxed)m_boxed).m_value;
+                return _value;
             }
         }
 
         /// <summary>
         /// Gets a value indicating whether this instance may be used concurrently from multiple threads.
         /// </summary>
-        internal LazyThreadSafetyMode Mode
+        internal LazyThreadSafetyMode? Mode
```

Comment:
Nit: All these forwarders can be written as one-liners:

`internal LazyThreadSafetyMode? Mode => LazyHelper.GetMode(_state);`
`internal bool IsValueFaulted => LazyHelper.GetIsValueFaulted(_state);`
`public bool IsValueCreated => _state == null;`